package.json

```diff
@@ -9,6 +9,7 @@
     "lib"
   ],
   "scripts": {
+    "precommit": "npm run test",
```

Comment:
Linting is *really* fast, so it will take just a little longer. It's mainly to prevent people that haven't set up xo linting in their editor, and forget to run it themselves.
We do that in next.js https://github.com/zeit/next.js/blob/master/package.json#L35